js/anattadesign/awesomecheckout/opcheckout.js

```diff
@@ -1763,3 +1763,21 @@ jQuery.validator.addClassRules({
 		creditcard2: ''
 	}
 });
+
+// Additions to support MasterCard series 2
+var cc2 = jQuery.validator.methods['creditcard2'];
+jQuery.validator.addMethod('creditcard2', function (value, element, param) {
+
+    var cardName = param;
+
+    if (typeof(cardName) !== 'string' || typeof(cardName) == 'string' && cardName == '') {
```

Comment:
do you need the OR here?
`cardName = (typeof(cardName) == 'string' && cardName !== ' ') ? param : ccType(value)`